man/systemd.exec.xml

```diff
@@ -968,6 +968,31 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>BindPaths=</varname></term>
+        <term><varname>BindReadOnlyPaths=</varname></term>
```

Comment:
I know this is bikeshedding, but I'd call the second one BindPathsReadOnly. Somehow that seems easier to pronounce. Or maybe ReadOnlyBindPaths.

As a bonus, `strstr`s below could be replaced by `endswith` / `startswith` ;)